languages/python/config.json

```diff
@@ -20,14 +20,20 @@
         "uuid": "7ecd42c1-2ed5-487a-bc89-71c9cbad9b05",
         "concepts": ["tuples"],
         "prerequisites": [
-          "boleans",
+          "bools",
           "for-loops",
           "functions",
           "if-keyword",
           "in-keyword",
           "integers",
           "return-keyword"
         ]
+      },
+      {
+        "slug": "bools",
+        "uuid": "c050470d-ef22-4187-aaac-239e07955a4d",
+        "concepts": ["bool"],
```

Comment:
@ErikSchierboom Does the concept name need to match the exercise name (and pre-requisites on other exercises)?

Reply:
Seems addressed by prior review comment https://github.com/exercism/v3/pull/1952#discussion_r458060453